### PR TITLE
Add September 2024 newsletter

### PR DESCRIPTION
**Pull request by CollierCZ**
----
📚 Documentation preview 📚: https://writethedocs-www--2204.org.readthedocs.build/

### CODE REVIEW

File: docs/blog/newsletter-september-2024.rst [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+.. post:: September 04, 2024
+  :tags: newsletter
+
+##########################################
+Write the Docs Newsletter – September 2024
+##########################################
+
+Howdy, documentarians. I hope everything is going well in your part of the world. We're back from out regular August break and already setting records. So far, it's just heat records here, but maybe we'll set the record for most documentarians inspired this month.
+
+In conference news, the Atlantic conference is coming up this month on 22–24 September. It's virtual so there's still time to get a ticket and you don't have to worry about travel. See the `planned talks </conf/atlantic/2024/speakers/>`__ and then `grab a ticket </conf/atlantic/2024/tickets/>`__. And if seeing the list of talks gets you more inspired, the deadline for `talk proposals for the Australia conference </conf/australia/2024/cfp/>`__ is this Friday (6 September). So get your ideas in while you still can!
```
Comment (swapnilogale) [inline docs/blog/newsletter-september-2024.rst]: @CollierCZ can you add a line at the end that says, Write the Docs Australia talks will be announced by end of September so folks can start planning travel etc?

Comment (CollierCZ) [inline docs/blog/newsletter-september-2024.rst]:
```suggestion
In conference news, the Atlantic conference is coming up this month on 22–24 September. It's virtual so there's still time to get a ticket and you don't have to worry about travel. See the `planned talks </conf/atlantic/2024/speakers/>`__ and then `grab a ticket </conf/atlantic/2024/tickets/>`__. And if seeing the list of talks gets you more inspired, the deadline for `talk proposals for the Australia conference </conf/australia/2024/cfp/>`__ is this Friday (6 September). So get your ideas in while you still can! The speaker lineup will be announced by the end of September so you can start planning travel to the conference.
```

File: docs/blog/newsletter-september-2024.rst [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+* Use the `OpenAPI Specification (OAS) <https://www.openapis.org/>`_ to define APIs. The OAS provides a clear source of truth, allows customers and in-house developers to generate code from OAS files, and supports good documentation.
+
+* Require descriptive, meaningful field names that do not use abbreviations or acronyms. For example, use "maximumQuantityToShip" instead of "maxQtySh". Pay special attention to Boolean fields to avoid unintuitive settings, such as `"cancel": false` actually meaning "yes, cancel this." Also, require consistent endpoint paths, ID names across endpoints, and patterns for things like pagination.
+
+* Implement `contract testing <https://smartbear.com/blog/api-contract-testing-for-a-design-first-world/>`_ to ensure that all examples work as documented. Establish a process for adding examples to the tests as needed.
+
+* Adopt an `API-first <https://www.youtube.com/watch?v=ODjX_3tGeeM&t=446>`_ approach: Designing APIs up front saves time and effort and ensures that the documentation is always up-to-date.
```
Comment (CollierCZ) [inline docs/blog/newsletter-september-2024.rst]:
```suggestion
* Adopt an `API-first <https://www.youtube.com/watch?v=ODjX_3tGeeM&t=446>`_ approach: Designing APIs up front saves time and effort and ensures that the documentation is always up to date.
```

File: docs/blog/newsletter-september-2024.rst [hunk 3]
```diff
@@ -0,0 +1,115 @@
+:og:image: https://www.writethedocs.org/_static/logo-opengraph.png
+
+.. post:: September 04, 2024
```
Comment (CollierCZ) [inline docs/blog/newsletter-september-2024.rst]:
```suggestion
.. post:: September 05, 2024
```

**Review by plaindocs — APPROVED** [review]
A+good

**Pull request merged**